packages/ckeditor5-ui/src/toolbar/toolbarview.js

```diff
@@ -302,7 +302,7 @@ export default class ToolbarView extends View {
 				console.warn( attachLinkToDocumentation(
 					'toolbarview-item-unavailable: The requested toolbar item is unavailable.' ), { name } );
 			}
-		} );
+		} ).filter( item => item !== undefined ) );
```

Comment:
Is there any test for this one? Why is it needed anyway?

Reply:
In case factory has no matching button then an undefined is returned (from the callback above). It happens if you misspell button name in the configuration.

There's at least one existing test that will fail if it's not handled correctly: https://github.com/ckeditor/ckeditor5/blob/e2befc89a888eb86366eda9fd5d4960a4e0c04dd/packages/ckeditor5-ui/tests/toolbar/toolbarview.js#L411-L426